### PR TITLE
Update Developers Guide

### PR DESCRIPTION
**Pull request by lokidoki102**
Updated the manual testing section for edit commands

### CODE REVIEW

**Review by redcolorbicycle — APPROVED** [review]

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -973,46 +973,57 @@ testers are expected to do more *exploratory* testing.
...
+   **Expected:** An error message indicating that "This person already exists in EstateEase."<br><br>
+
+1. **Invalid person type**<br>
+   1. **Test case** `editSeller 1 n/Jessi Oliverson`<br>
+   **Expected:** An error message indicating that "The person you are trying to edit is not a seller."<br><br>
+
 1. **Invalid INDEX**<br>
-   **Test case (Invalid INDEX):** `editSeller 0 p/87654321`<br>
-   Expected: An error messsage indicating that the command has invalid format.
+   1. **Test case (Invalid INDEX):** `editSeller 0 p/87654321`<br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:1005]: Perhaps Invalid Index can be removed

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -973,46 +973,57 @@ testers are expected to do more *exploratory* testing.
...
 
+### Editing Buyer Details
+
+**Prerequisites:**
+- There is a buyer named "James Cook".
+- For each `editBuyer` command execution in this manual testing, execute the `find` command on the targeted
+  buyer/seller to ensure that this person is displayed as the first person in the list.
+
+1. **Invalid budget value**<br>
+    1. **Test case:** `editBuyer 1 budget/-1`<br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:1017]: I think can another testcase with budget 0

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -973,47 +973,74 @@ testers are expected to do more *exploratory* testing.
...
-1. **Wrong type**<br>
-   **Test case:** `editSeller 2 n/Jessi Oliverson`<br>
-   Expected: An error message indicating that "The person you are trying to edit is not a seller."
-1. **Successful edit**<br>
-   **Test case:** `editSeller 1 n/Jessi Oliverson`<br>
-   Expected: The name "aaaaaaa" is edited to "Jessi Oliverson".
-   The updated details of the edited seller will also be shown in the response box.
+
+   1. **Test case:** `editSeller 1 n/John Zy`<br>
+   **Expected:** An error message indicating that "This person already exists in EstateEase."<br><br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:999]: General error message should be fine

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -973,47 +973,74 @@ testers are expected to do more *exploratory* testing.
...
+
+1. **Invalid person type**<br>
+
+   1. **Test case** `editSeller 1 n/Jessi Oliverson`<br>
+   **Expected:** An error message indicating that "The person you are trying to edit is not a seller."<br><br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:1004]: Same as above

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -973,47 +973,76 @@ testers are expected to do more *exploratory* testing.
...
+- For each `editBuyer` command execution in this manual testing, execute the `find` command on the targeted
+  buyer/seller to ensure that this person is displayed as the first person in the list.
+
+1. **Invalid budget value**<br>
+
+    1. **Test case:** `editBuyer 1 budget/-1`<br>
+       **Expected:** An error message will be shown, indicating that the budget should be a positive number."<br><br>
+
+    1. **Test case:** `editBuyer 1 budget/0`<br>
+       **Expected:** An error message will be shown, indicating that "Budget should be a positive number."<br><br>
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:1032]: Perhaps remove "

**Review by felixchanyy — APPROVED** [review]
LGTM

**Pull request merged**